ext/dom/tests/DOMDocument_getElementsByTagNameNS_match_any_namespace.phpt

```diff
@@ -0,0 +1,82 @@
+--TEST--
+DOMDocument::getElementsByTagNameNS() match any namespace
+--EXTENSIONS--
+dom
+--FILE--
+<?php
+
+/* Sample document taken from https://www.php.net/manual/en/domdocument.getelementsbytagname.php */
+$xml = <<<EOD
+<?xml version="1.0" ?>
+<chapter xmlns:xi="http://www.w3.org/2001/XInclude">
+<title>Books of the other guy..</title>
+<para>
+ <xi:include href="book.xml">
+  <xi:fallback>
+   <error>xinclude: book.xml not found</error>
+  </xi:fallback>
+ </xi:include>
+ <include>
+  This is another namespace
+ </include>
+</para>
+</chapter>
+EOD;
+$dom = new DOMDocument;
+
+// load the XML string defined above
+$dom->loadXML($xml);
+
+function test($namespace, $local) {
+    global $dom;
+    $namespace_str = $namespace !== NULL ? "'$namespace'" : "null";
```

Comment:
I'm confused by what this line does ._.

Reply:
If $namespace is NULL, I want to display `null`, otherwise (so a normal string) I want to display `'string'`.
It's just a stylistic choice so I don't see `'null'` in the output as that can be confusing.

Reply:
Right, okay makes sense. :)